src/pages/components/number-input/usage.mdx

```diff
@@ -18,14 +18,49 @@ controls for incrementally increasing or decreasing the value.
 <AnchorLink>Overview</AnchorLink>
 <AnchorLink>Live demo</AnchorLink>
 <AnchorLink>Formatting</AnchorLink>
+<AnchorLink>Content</AnchorLink>
+<AnchorLink>Behaviors</AnchorLink>
+<AnchorLink>Related</AnchorLink>
+<AnchorLink>References</AnchorLink>
 <AnchorLink>Feedback</AnchorLink>
 
 </AnchorLinks>
 
 ## Overview
 
-Number inputs are similar to text inputs, but contain controls used to increase
-or decrease an incremental value.
+Number inputs are similar to text inputs, but used to specify a numeric value.
+Number input lets users enter a numeric value and incrementally increase or
+decrease the value with a two-segment control. Number input reduces input
+efforts for adjusting small values.
+
+<Row>
+<Column colLg={8}>
+
+![number input image](images/number-input-usage-1.png)
+
+</Column>
+</Row>
+
+### When to use
+
+- Enabling the user to type a number in the field
+- Adjusting small values when increasing or decreasing them requires only a few
+  clicks
+- When users may not know exact values and only want to change the values that
+  are relative to its current state
+
+### When not to use
+
+- It is best practice not to use number input if large value changes are
+  expected. For examples, when the value sets from 1 to 30, number input is not
+  a great option. In this case, use
```

Comment:
Can you explain why it is not a great option? Is it because it would require many clicks to reach 30?